Classes/Emogrifier.php

```diff
@@ -1501,4 +1505,34 @@ private function getNodesToExclude(\DOMXPath $xPath)
 
         return $excludedNodes;
     }
+
+    /**
+     * Handle warnings in process() method
+     *
+     * @param int $errNo
+     * @param string $errStr
+     * @param string $errFile
+     * @param int $errLine
+     * @param array $errContext
+     *
+     * @return bool
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function errorHandler($errNo, $errStr, $errFile, $errLine, $errContext)
```

Comment:
No abrreviations ("err"), please. Actually, I think that we don't need the prefix anyway as the context already is given by the method.